packages/ui-library/src/components/radio/index.test.ts

```diff
@@ -99,4 +100,21 @@ describe('blr-radio', () => {
 
     expect(className).not.to.contain('error');
   });
+
+  it('has error Icon set to undefined', async () => {
+    const element = await fixture(
+      BlrRadioRenderFunction({
+        ...sampleParams,
+        hasHint: false,
+        hasError: false,
+        errorIcon: undefined,
+      })
+    );
+    const labelWrapper = querySelectorDeep('.label-wrapper', element.getRootNode() as HTMLElement);
+    const captionWrapper = querySelectorDeep('.caption-wraper', labelWrapper?.getRootNode() as HTMLElement);
+    const formCaption = querySelectorDeep('.blr-form-caption', captionWrapper?.getRootNode() as HTMLElement);
+
```

Comment:
This extra new line can be removed for the consistency purpose. Right now they are mixed up.